cfme/tests/cli/test_appliance_console.py

```diff
@@ -1720,6 +1730,73 @@ def test_appliance_console_vmdb_log_negative(temp_appliance_preconfig_funcscope)
     ap_console_tail.validate(wait="60s")
 
 
+@pytest.mark.ignore_stream("5.10")
+@pytest.mark.meta(automates=[1670138])
+@pytest.mark.meta(blockers=[BZ(1670138, forced_streams=["5.10"])])
+def test_appliance_overwrite_ssl_ipa(unconfigured_appliance, freeipa_provider):
```

Comment:
Keep ignore stream only if this feature is not going to be in 5.10 otherwise blocker with forced stream is ok.